_posts/0000-01-02-ridgwayk.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
You Text is boring, especially with all the time you have on your hands.  Be more creative in your comments. Winter is here. :-)

Reply:
ok, your changes look good...after all, I made them!